Dockerfile.dapper

```diff
@@ -1,15 +1,26 @@
-FROM golang:1.6
-RUN go get github.com/imikushin/trash
+FROM alpine:3.4
+# FROM arm64=armhf/alpine:3.4 arm=armhf/alpine:3.4
+
+ARG DAPPER_HOST_ARCH
+ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH}
+
+RUN apk -U add bash gcc musl-dev go git perl linux-headers openssl util-linux
+
+ENV GOPATH=/go PATH=/go/bin:${PATH}
+
+RUN go get github.com/rancher/trash
+
+
+ENV SHELL /bin/bash
+
 RUN go get github.com/golang/lint/golint
-RUN curl -sL https://get.docker.com/builds/Linux/x86_64/docker-1.9.1 > /usr/bin/docker && \
-    chmod +x /usr/bin/docker
-ENV PATH /go/bin:$PATH
-ENV DAPPER_SOURCE /go/src/github.com/rancher/rancher-dns
-ENV DAPPER_OUTPUT bin dist
-ENV DAPPER_DOCKER_SOCKET true
+
+ENV DAPPER_SOURCE=/go/src/github.com/rancher/rancher-dns
+ENV DAPPER_OUTPUT ./bin ./dist
 ENV DAPPER_ENV TAG REPO
-ENV GO15VENDOREXPERIMENT 1
 ENV TRASH_CACHE ${DAPPER_SOURCE}/.trash-cache
+
 WORKDIR ${DAPPER_SOURCE}
+
 ENTRYPOINT ["./scripts/entry"]
-CMD ["ci"]
+CMD ["release"]
```

Comment:
The image shouldn't default to running `release`.  Please have a script called `ci` and then if there is no difference for `release`, just make it a one liner that calls `ci`